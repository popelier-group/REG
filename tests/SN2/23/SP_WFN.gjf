# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.30649900   -0.00006800    0.00014800
 H                 -0.99963100   -0.87934200   -0.54768900
 H                 -0.99954100    0.91400100   -0.48736600
 H                 -0.99969800   -0.03490700    1.03555900
 Br                 1.65489700    0.00001100   -0.00001700
 Cl                -3.20003000    0.00000500   -0.00002600

frame.wfn
