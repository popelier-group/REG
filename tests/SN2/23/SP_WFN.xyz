
6

C       -1.123228    0.112195    0.365607
H       -0.617566   -0.488808    1.107575
H       -1.161246   -0.373943   -0.598572
H       -0.719328    1.112453    0.304237
Br        1.680177   -0.167636   -0.546798
Cl       -2.915805    0.290846    0.948885
