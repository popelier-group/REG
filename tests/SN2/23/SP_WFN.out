 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.23/Gau-75529.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.23/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     75530.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.3065   -0.00007   0.00015 
 H                    -0.99963  -0.87934  -0.54769 
 H                    -0.99954   0.914    -0.48737 
 H                    -0.9997   -0.03491   1.03556 
 Br                    1.6549    0.00001  -0.00002 
 Cl                   -3.20003   0.00001  -0.00003 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.306499   -0.000068    0.000148
      2          1           0       -0.999631   -0.879342   -0.547689
      3          1           0       -0.999541    0.914001   -0.487366
      4          1           0       -0.999698   -0.034907    1.035559
      5         35           0        1.654897    0.000011   -0.000017
      6         17           0       -3.200030    0.000005   -0.000026
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.080470   0.000000
     3  H    1.080470   1.794357   0.000000
     4  H    1.080471   1.794365   1.794360   0.000000
     5  Br   2.961396   2.849513   2.849373   2.849651   0.000000
     6  Cl   1.893531   2.432065   2.432086   2.432101   4.854927
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.123228    0.112195    0.365607
      2          1           0       -0.617566   -0.488808    1.107575
      3          1           0       -1.161246   -0.373943   -0.598572
      4          1           0       -0.719328    1.112453    0.304237
      5         35           0        1.680177   -0.167636   -0.546798
      6         17           0       -2.915805    0.290846    0.948885
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.7444420           0.8432159           0.8432159
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.1811143430 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.76D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40012671     A.U. after   11 cycles
            NFock= 11  Conv=0.79D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55831-101.40077 -62.21145 -56.02927 -56.02915
 Alpha  occ. eigenvalues --  -56.02915 -10.07995  -9.31485  -8.42027  -7.07837
 Alpha  occ. eigenvalues --   -7.07022  -7.07022  -6.24348  -6.24293  -6.24293
 Alpha  occ. eigenvalues --   -2.35585  -2.35567  -2.35567  -2.35523  -2.35523
 Alpha  occ. eigenvalues --   -0.69636  -0.54171  -0.49106  -0.30308  -0.30308
 Alpha  occ. eigenvalues --   -0.26050  -0.16962  -0.16962  -0.05650  -0.05611
 Alpha  occ. eigenvalues --   -0.05610
 Alpha virt. eigenvalues --    0.11162   0.12480   0.15036   0.15036   0.17448
 Alpha virt. eigenvalues --    0.18210   0.19584   0.19584   0.20830   0.20830
 Alpha virt. eigenvalues --    0.21686   0.22630   0.22630   0.24850   0.28719
 Alpha virt. eigenvalues --    0.31115   0.31115   0.37122   0.40532   0.53871
 Alpha virt. eigenvalues --    0.53872   0.55560   0.58933   0.58933   0.58965
 Alpha virt. eigenvalues --    0.62019   0.62019   0.66923   0.76965   0.76965
 Alpha virt. eigenvalues --    0.79622   0.83972   0.87588   0.87589   0.88866
 Alpha virt. eigenvalues --    0.88867   0.96110   0.98156   0.98157   1.01953
 Alpha virt. eigenvalues --    1.01953   1.03925   1.13159   1.26446   1.26447
 Alpha virt. eigenvalues --    1.26588   1.56762   1.67694   1.67694   1.86009
 Alpha virt. eigenvalues --    1.86010   1.87421   2.06789   2.20872   2.20874
 Alpha virt. eigenvalues --    2.27877   2.27878   2.31894   2.45698   2.45698
 Alpha virt. eigenvalues --    2.57407   2.60960   2.64238   2.64238   2.68536
 Alpha virt. eigenvalues --    2.68536   3.14611   3.26687   3.87206   3.87206
 Alpha virt. eigenvalues --    4.39348   4.39349   4.40266   4.40266   4.54887
 Alpha virt. eigenvalues --    6.77132   7.80152   7.80159   7.86929   9.84181
 Alpha virt. eigenvalues --   24.04058  25.99199  25.99199  26.48975  48.08380
 Alpha virt. eigenvalues --  215.82731 290.01024 290.01025 290.072781020.84437
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.669146   0.409887   0.409884   0.409892  -0.065765  -0.200129
     2  H    0.409887   0.520650  -0.038464  -0.038470  -0.003951  -0.042460
     3  H    0.409884  -0.038464   0.520633  -0.038468  -0.003956  -0.042446
     4  H    0.409892  -0.038470  -0.038468   0.520658  -0.003940  -0.042468
     5  Br  -0.065765  -0.003951  -0.003956  -0.003940  35.918393   0.060973
     6  Cl  -0.200129  -0.042460  -0.042446  -0.042468   0.060973  17.310283
 Mulliken charges:
               1
     1  C   -0.632916
     2  H    0.192808
     3  H    0.192817
     4  H    0.192796
     5  Br  -0.901753
     6  Cl  -0.043752
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.054495
     5  Br  -0.901753
     6  Cl  -0.043752
 Electronic spatial extent (au):  <R**2>=           1141.8121
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -3.5132    Y=              0.3508    Z=              1.1435  Tot=              3.7112
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -72.4176   YY=            -44.6306   ZZ=            -47.3237
   XY=              2.7997   XZ=              9.1336   YZ=             -0.9111
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -17.6270   YY=             10.1600   ZZ=              7.4670
   XY=              2.7997   XZ=              9.1336   YZ=             -0.9111
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             46.1540  YYY=             -1.6927  ZZZ=             -9.0701  XYY=              8.8157
  XXY=             -2.9823  XXZ=             -9.5684  XZZ=             10.5707  YZZ=             -1.8632
  YYZ=             -2.6902  XYZ=              0.4273
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1373.6755 YYYY=            -87.0398 ZZZZ=           -202.0583 XXXY=             70.7128
 XXXZ=            230.3170 YYYX=             58.4631 YYYZ=            -18.9870 ZZZX=            196.2922
 ZZZY=            -20.1575 XXYY=           -225.4552 XXZZ=           -255.2050 YYZZ=            -48.1675
 XXYZ=             -9.8382 YYXZ=             64.3721 ZZXY=             21.3368
 N-N= 1.711811143430D+02 E-N=-7.695806494164D+03  KE= 3.070047868570D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE366\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -1.306499,-0.000068,0.000148\H,0,-0.999631,-0.879342,-0.547689\H,0,-0.
 999541,0.914001,-0.487366\H,0,-0.999698,-0.034907,1.035559\Br,0,1.6548
 97,0.000011,-0.000017\Cl,0,-3.20003,0.000005,-0.000026\\Version=ES64L-
 G16RevC.01\State=1-A\HF=-3074.4001267\RMSD=7.932e-09\Dipole=-1.4601097
 ,-0.0000557,0.0001154\Quadrupole=-15.5227515,7.7613773,7.7613742,0.000
 0308,-0.0000987,0.0000113\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


    WHAT WE HAVE LEARNED
 IS LIKE A HANDFUL OF EARTH;
  WHAT WE HAVE YET TO LEARN
   IS LIKE THE WHOLE WORLD.

         -- AVVAYIAR
 Job cpu time:       0 days  0 hours  0 minutes 28.0 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.5 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:52:11 2020.
